.yamato/_triggers.yml

```diff
@@ -0,0 +1,43 @@
+{% metadata_file .yamato/project.metafile %}
+---
+
+# Run all relevant tasks when a pull request targets the develop / main or a sample branch
+pull_request_trigger:
+  name: Pull Request Trigger (main, develop, & sample branches)
+  dependencies:
+{% for project in projects -%}
+{% for editor in project.test_editors -%}
+{% for platform in test_platforms -%}
+    - .yamato/build.yml#build_{{ project.name }}_{{ editor }}_{{ platform.name }}
```

Comment:
This is the line that I feel is redundant. I'll investigate in a future PR, after integrating into develop.